src/main/java/jnr/ffi/provider/jffi/BufferParameterStrategy.java

```diff
@@ -21,27 +21,94 @@
 import com.kenai.jffi.MemoryIO;
 import com.kenai.jffi.ObjectParameterType;
 
-import java.nio.Buffer;
+import java.nio.*;
```

Comment:
Please use explicit imports (I know we are not consistent about this but we try).

Reply:
Ah that's IntelliJ doing its thing. I'll change the settings for this project.